fix test on macos big sur

# Some tests are failing on mac os after upgrading to big sur

```
ERROR: Unable to load library 'CoreFoundation':
dlopen(libCoreFoundation.dylib, 9): image not found
dlopen(libCoreFoundation.dylib, 9): image not found
Native library (darwin/libCoreFoundation.dylib) not found in resource path ([file:/Users/vince/.gradle/caches/6.3/workerMain/gradle-worker.jar, file:/Users/vince/workspace/perso/final/opa-idea-plugin/build/[…]
assertj-core-3.16.1.jar, file:/Users/vince/.gradle/caches/modules-2/files-2.1/com.github.kittinunf.result/result/2.2.0/5d0e46da3a4c8f22d64ee391fd7aea13ec2e6d30/result-2.2.0.jar])
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:302)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:455)
	at com.sun.jna.Library$Handler.<init>(Library.java:192)
	at com.sun.jna.Native.load(Native.java:596)
	at com.sun.jna.Native.load(Native.java:570)
	at com.intellij.util.text.DateFormatUtil.getMacFormats(DateFormatUtil.java:363)
	at com.intellij.util.text.DateFormatUtil.getDateTimeFormats(DateFormatUtil.java:303)
	at com.intellij.util.text.DateFormatUtil.<clinit>(DateFormatUtil.java:46)
	at com.intellij.execution.testframework.sm.runner.ui.SMTestRunnerResultsForm.onTestingStarted(SMTestRunnerResultsForm.java:239)
	at com.intellij.execution.testframework.sm.runner.GeneralTestEventsProcessor.fireOnTestingStarted(GeneralTestEventsProcessor.java:145)
	at com.intellij.execution.testframework.sm.runner.GeneralIdBasedToSMTRunnerEventsConvertor.onStartTesting(GeneralIdBasedToSMTRunnerEventsConvertor.java:44)
	at com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter.startTesting(OutputToGeneralTestEventsConverter.java:344)
	at com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil.attachEventsProcessors(SMTestRunnerConnectionUtil.java:202)
	at com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil.access$000(SMTestRunnerConnectionUtil.java:36)
	at com.intellij.execution.testframework.sm.SMTestRunnerConnectionUtil$1.onAttachToProcess(SMTestRunnerConnectionUtil.java:133)
	at com.intellij.execution.testframework.sm.runner.ui.SMTRunnerConsoleView.attachToProcess(SMTRunnerConsoleView.java:96)
	at org.openpolicyagent.ideaplugin.ide.runconfig.test.OpaTestRunProfileState.execute(OpaTestRunProfileState.kt:69)
	at org.openpolicyagent.ideaplugin.ide.runconfig.RunConfigurationTestBase.execute(RunConfigurationTestBase.kt:54)
	at org.openpolicyagent.ideaplugin.ide.runconfig.test.OpaTestRunConfigurationBase.executeAndGetTestRoot(OpaTestRunConfigurationBase.kt:42)
	at org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest.test runner all test error(TestRunConfigurationExecutionOpaTest.kt:193)
[…]



org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest > test runner all test error FAILED
    com.intellij.util.lang.CompoundRuntimeException at UsefulTestCase.java:411
        Caused by: com.intellij.testFramework.LoggedErrorProcessor$TestLoggerAssertionError at TestRunConfigurationExecutionOpaTest.kt:193
            Caused by: java.lang.UnsatisfiedLinkError at TestRunConfigurationExecutionOpaTest.kt:193

org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest > test runner with test error pass and fail FAILED
    com.intellij.util.lang.CompoundRuntimeException at UsefulTestCase.java:411
        Caused by: java.lang.NoClassDefFoundError at TestRunConfigurationExecutionOpaTest.kt:132

org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest > test runner with test pass fail and error FAILED
    com.intellij.util.lang.CompoundRuntimeException at UsefulTestCase.java:411
        Caused by: java.lang.NoClassDefFoundError at TestRunConfigurationExecutionOpaTest.kt:70

org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest > test runner with all tests fail FAILED
    com.intellij.util.lang.CompoundRuntimeException at UsefulTestCase.java:411
        Caused by: java.lang.NoClassDefFoundError at TestRunConfigurationExecutionOpaTest.kt:253

org.openpolicyagent.ideaplugin.ide.runconfig.test.TestRunConfigurationExecutionOpaTest > test runner all test pass FAILED
    com.intellij.util.lang.CompoundRuntimeException at UsefulTestCase.java:411
        Caused by: java.lang.NoClassDefFoundError at TestRunConfigurationExecutionOpaTest.kt:311
```

--- a/gradle-201.properties
+++ b/gradle-201.properties
@@ -4,8 +4,8 @@
 #
 
 # if you change the version of ide, also change psiViewerPluginVersion accordingly (cf https://plugins.jetbrains.com/plugin/227-psiviewer/versions)
-ideaVersion=IC-2020.1.2
-pycharmCommunityVersion=PC-2021.2
+ideaVersion=IC-2020.1.4
+pycharmCommunityVersion=PC-2020.2.5
 psiViewerPluginVersion=201.6251.22-EAP-SNAPSHOT.3
 
 # see https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for more information

--- a/gradle-202.properties
+++ b/gradle-202.properties
@@ -4,8 +4,8 @@
 #
 
 # if you change the version of ide, also change psiViewerPluginVersion accordingly (cf https://plugins.jetbrains.com/plugin/227-psiviewer/versions)
-ideaVersion=IC-2020.2
-pycharmCommunityVersion=PC-2020.2
+ideaVersion=IC-2020.2.4
+pycharmCommunityVersion=PC-2020.2.4
 psiViewerPluginVersion=202-SNAPSHOT.3
 
 # see https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for more information

--- a/src/test/kotlin/org/openpolicyagent/ideaplugin/lang/RegoParsingTestCaseBase.kt
+++ b/src/test/kotlin/org/openpolicyagent/ideaplugin/lang/RegoParsingTestCaseBase.kt
@@ -5,12 +5,14 @@
 
 package org.openpolicyagent.ideaplugin.lang
 
+import com.intellij.lang.LanguageBraceMatching
 import com.intellij.testFramework.ParsingTestCase
 import com.intellij.util.EnvironmentUtil
 import org.apache.commons.io.IOUtils
 import org.openpolicyagent.ideaplugin.OpaTestBase
 import org.openpolicyagent.ideaplugin.OpaTestCase
 import org.openpolicyagent.ideaplugin.OpaTestCase.Companion.testResourcesPath
+import org.openpolicyagent.ideaplugin.ide.typing.RegoBraceMatcher
 import org.openpolicyagent.ideaplugin.lang.parser.RegoParserDefinition
 import org.openpolicyagent.ideaplugin.opa.tool.OpaBaseTool.Companion.opaBinary
 
@@ -21,6 +23,12 @@ abstract class RegoParsingTestCaseBase() : ParsingTestCase(
     true,
     RegoParserDefinition()
 ), OpaTestCase {
+
+    override fun setUp() {
+        super.setUp()
+        // fix issue #71
+        addExplicitExtension(LanguageBraceMatching.INSTANCE, RegoLanguage, RegoBraceMatcher())
+    }
 
     override fun getTestDataPath(): String = testResourcesPath
 

